### ISSUE TITLE
hourglass.pth and a user warning

### ISSUE THREAD

**Issue by yosunpeng** [ouc-ocean-group/RRNet#1]
# Error: No such files : './hourglass.pth'
get pre-trained weight from [weights1](https://github.com/anibali/pytorch-stacked-hourglass/releases) or [weights2](https://drive.google.com/drive/folders/0B63t5HSgY4SQTzNQWWplelF3eEk).
rename .pth file name to hourglass.pth and put it at correct path. Done.
# User Warning
`
/usr/lib/python3.8/site-packages/torch/optim/lr_scheduler.py:95: UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of   the learning rate schedule.See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
`
go to operator/rrnet_operator.py , around line 117:
` self.lr_sch.step()`
delete it and move it to around line 140:
after `self.optimizer.step()`

the order should be : 

self.optimizer.step() 
self.lr_sch.step()

Two simple problems, just for saving others' time.
